docs/serverless/AI-for-security/ai-assistant.mdx

```diff
@@ -99,6 +99,9 @@ Use these features to adjust and act on your conversations with AI Assistant:
         Be sure to specify which language you'd like AI Assistant to use when writing a query. For example: "Can you generate an Event Query Language query to find four failed logins followed by a successful login?"
         </DocCallOut>
 
+<DocCallOut title="Tip">
+AI Assistant can remember particular information you tell it to remember. For example, you could tell it: "When anwering any question about srv-win-s1-rsa or an alert that references it, mention that this host is in the New York data center". This will cause it to remember the detail you highlighted.
+</DocCallOut>
```

Comment:
The stacked callouts look a little jumbly. Consolidate into a single callout called "Tips"?

<img width="601" alt="image" src="https://github.com/user-attachments/assets/96a84b3b-8973-4d3d-bd8c-e3e65e3f7b06">